nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core/Run/ApiModel/DependencyGroup.cs

```diff
@@ -1,8 +1,68 @@
+using System.Collections.Immutable;
+using System.IO.Enumeration;
+
 namespace NuGetUpdater.Core.Run.ApiModel;
 
 public record DependencyGroup
 {
     public required string Name { get; init; }
     public string? AppliesTo { get; init; }
+
+    // TODO: make more strongly typed, but currently this seems to be:
+    //   "patterns" => string[] where each element is a wildcard name pattern
+    //   "exclude-patterns"=> string[] where each element is a wildcard name pattern
+    //   "dependency-type" => production|development // not used for nuget?
```

Comment:
This is not for now, but does use of `PrivateAssets="All"` equivalent to `development` in NuGet?